KsApi/models/graphql/adapters/Comment+GraphCommentTests.swift

```diff
@@ -0,0 +1,20 @@
+@testable import KsApi
+import XCTest
+
+final class Comment_GraphCommentTests: XCTestCase {
+  func test() {
```

Comment:
Do we need to nest `test` functions here?

Reply:
Sorry, not sure what you mean?

Reply:
I commented on the wrong line, but I believe right under this `func test()` is another `func test()`.

Reply:
oh lol yes, copy paste error, thanks!